Makefile

```diff
@@ -8,7 +8,7 @@ build: test
 
 .PHONY: image
 image: build
-	docker build -t "quay.io/numaio/numaflow-source/nats-source-go:v0.5.0" --target nats-source .
+	docker build -t "quay.io/numaio/numaflow-source/nats-source-go:v0.5.10" --target nats-source .
```

Comment:
did this get updated because you were updating the image tag each time as you pushed a new one? (even though starting from v0.5.0 was arbitrary)?

Reply:
I was testing my changes using my quick start. So every time I made a change, I pushed a new image. I went through a couple of iterations hence you can see the number increased accordingly.